clustering/jgroups/extension/src/main/java/org/jboss/as/clustering/jgroups/logging/JGroupsLogger.java

```diff
@@ -164,4 +164,8 @@ public interface JGroupsLogger extends BasicLogger {
     @LogMessage(level = WARN)
     @Message(id = 31, value = "Ignoring unrecognized %s property: %s")
     void unrecognizedProtocolProperty(String protocol, String property);
+
+    @LogMessage(level = WARN)
+    @Message(id = 32, value = "Attribute '%s' of element '%s' is deprecated and will be ignored.")
+    void deprecatedAttribute(String attribute, String element);
```

Comment:
Let's call this 'ignoredAttribute'.

Moreover, let's sync the wording with -https://github.com/wildfly/wildfly/pull/15572/files#diff-d85e58deccc37aa65a19826095d1b918a35c05b7509e18144d3142ce57745a2fR115 - s/deprecated/is no longer supported

Reply:
Done